presto-main/src/main/java/com/facebook/presto/event/query/QueryMonitor.java

```diff
@@ -126,7 +126,7 @@ public void queryCreatedEvent(QueryInfo queryInfo)
                                 queryInfo.getSession().getSource(),
                                 queryInfo.getSession().getCatalog(),
                                 queryInfo.getSession().getSchema(),
-                                queryInfo.getResourceGroupName(),
+                                queryInfo.getResourceGroupId(),
```

Comment:
This commit title would be sufficient (no need for full message):
Replace resourceGroupName with resourceGroupId in QueryInfo
Also, squash the last commit into this one - we prefer to not have commits where Presto is broken.